.github/workflows/build-executable.yml

```diff
@@ -0,0 +1,32 @@
+name: Bulid Executable
+
+on:
+  workflow_call:
+    inputs:
+      release-tag:
+        type: string
+        description: 'Release tag'
+        required: true
+
+jobs:
+
+  standalone-executable:
+    name: Create a standalone linux executable
+    runs-on: ubuntu-24.04
```

Comment:
The Ubuntu version will decide which glibc version we are supporting. We probably should use Ubuntu 22.04 instead.